aten/src/ATen/functorch/BatchRulesModules.cpp

```diff
@@ -384,15 +384,15 @@ TORCH_LIBRARY_IMPL(aten, FuncTorchBatched, m) {
   EXISTING_BDIM_ALL_BOXED(reflection_pad2d_backward);
   EXISTING_BDIM_ALL_BOXED(reflection_pad3d_backward);
 
-  UPSAMPLE_BATCH(upsample_bicubic2d);
-  UPSAMPLE_BATCH(upsample_bilinear2d);
-  UPSAMPLE_BATCH(upsample_linear1d);
-  UPSAMPLE_BATCH(upsample_nearest1d);
-  UPSAMPLE_BATCH(upsample_nearest2d);
-  UPSAMPLE_BATCH(upsample_nearest3d);
-  UPSAMPLE_BATCH(upsample_trilinear3d);
-  UPSAMPLE_BATCH(_upsample_bilinear2d_aa);
-  UPSAMPLE_BATCH(_upsample_bicubic2d_aa);
+  EXISTING_BDIM(upsample_bicubic2d);
+  EXISTING_BDIM(upsample_bilinear2d);
+  EXISTING_BDIM(upsample_linear1d);
+  EXISTING_BDIM(upsample_nearest1d);
+  EXISTING_BDIM(upsample_nearest2d);
+  EXISTING_BDIM(upsample_nearest3d);
+  EXISTING_BDIM(upsample_trilinear3d);
+  EXISTING_BDIM(_upsample_bilinear2d_aa);
+  EXISTING_BDIM(_upsample_bicubic2d_aa);
```

Comment:
What's the difference between UPSAMPLE_BATCH and EXISTING_BDIM?

Reply:
Lol I see it now. We should delete the UPSAMPLE_BATCH macro, there aren't any of those anymore, right?